frontend/public/models/yaml-templates.ts

```diff
@@ -801,4 +801,16 @@ spec:
   machineSelector:
     matchLabels:
       node-role.kubernetes.io/master: ""
+`).setIn([referenceForModel(k8sModels.ConsoleNotificationModel), 'default'], `
+apiVersion: console.openshift.io/v1
+kind: ConsoleNotification
+metadata:
+  name: example
+spec:
+  notification:
+    text: This is an example notification message with an optional link.
+    link:
+      href: 'http://www.example.com'
+      text: Optional link text
+  scope: Cluster
```

Comment:
is it always cluster wide? in that case do we need to provide it? if its not, maybe add a non cluster scope example ?

Reply:
Good catch, @ohadlevy.  I've removed `scope` as notifications are always cluster-wide.